resources/views/dayarchive/index.blade.php

```diff
@@ -9,67 +9,109 @@
-            csv.zip and the csv.zip.sha1 files in the current folder, doing
-            sha1sum -c sor-global-2023-09-25-full.csv.zip.sha1 should show OK if the
-            sor-global-2023-09-25-full.csv.zip file has been correctly downloaded.
-        </div>
-    </div>
+    @if($platform)
+        <h1 class="ecl-page-header__title ecl-u-type-heading-1 ecl-u-mb-l">Daily Archives for {{ $platform->name }}</h1>
+    @else
+        <h1 class="ecl-page-header__title ecl-u-type-heading-1 ecl-u-mb-l">Data Download</h1>
+    @endif
 
     <div class="ecl-row ecl-u-mt-l">
-        <div class="ecl-col-l-6">
-            @if($platform)
-                <h1 class="ecl-page-header__title ecl-u-type-heading-1 ecl-u-mb-l">Daily Archives for {{ $platform->name }}</h1>
-            @else
-                <h1 class="ecl-page-header__title ecl-u-type-heading-1 ecl-u-mb-l">Data Download</h1>
-            @endif
+        <div class="ecl-col-l-8">
+            <p class="ecl-u-type-paragraph">
+                On this page, you can download zipped .csv files containing the daily submissions of statements of
+                reasons,
+                either for all platforms collectively or for each platform individually. The files are provided in
+                full and light versions.<br/>
+                <br/>
+                Full archive files contain all the public data points of each individual statement of reasons
+                submitted on a given day. That is, each file contains the entire attribute schema of the database.<br/>
+                <br/>
+
+
+                <a href="{{ route('page.show', ['faq']) }}">Read more about the Full and light version of the archive in
+                    the FAQ</a><br/><br/>
+
+
+                <a href="{{ route('page.show', ['faq']) }}">Read more about the archive format and the SHA1 in the
+                    FAQ</a>
+
+            </p>
+
+        </div>
+        <div class="ecl-col-l-4">
+
+            <div class="ecl-media-container">
+                <figure class="ecl-media-container__figure">
+                    <div class="ecl-media-container__caption">
 
+                        <picture class="ecl-picture ecl-media-container__picture"><img
+                                class="ecl-media-container__media"
+                                src="https://dsa-images-disk.s3.eu-central-1.amazonaws.com/dsa-image-2.jpeg"
+                                alt="Digital Services Act Logo"/></picture>
+
+
+                    </div>
+
+
+                </figure>
+
+
+            </div>
         </div>
 
-        <div class="ecl-col-l-6">
-            <form method="get" id="platform">
+    </div>
+    <form method="get" id="platform">
+        <div class="ecl-row ecl-u-mt-l" style="border-width: 50px">
+
+            <div class="ecl-col-l-2">
+
+                <x-ecl.datepicker label="From" id="created_at_start" justlabel="true"
```

Comment:
are you filtering on created at on the day archive model? created_at_end created_at_start? I would filter just on the "date" field.

Reply:
this was a dummy. We are now using date